internal/controller/isbservicerollout_controller.go

```diff
@@ -261,6 +269,16 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 
 	} else {
 		// Object already exists
+		// copy rollout annotations and labels to child resource
+		for val, key := range isbServiceRollout.Spec.InterStepBufferService.Annotations {
+			existingISBServiceDef.Annotations[val] = key
+		}
+		newISBServiceDef.Annotations = existingISBServiceDef.Annotations
+		for val, key := range isbServiceRollout.Spec.InterStepBufferService.Labels {
+			existingISBServiceDef.Labels[val] = key
```

Comment:
so, "existingISBServiceDef" is supposed to mean the isbsvc that exists in Kubernetes, whereas "newISBServiceDef" is supposed to mean the definition we intend to write. We don't want to update the "existingISBServiceDef" (same applies to all rollouts)

Reply:
Ack, I can copy the old labels/annotations first and handle them the same way I handle the case where the resource doesn't exist:
```newISBServiceDef.Annotations = existingISBServiceDef.Annotations
		for val, key := range isbServiceRollout.Spec.InterStepBufferService.Annotations {
			newISBServiceDef.Annotations[val] = key
		}
		newISBServiceDef.Labels = existingISBServiceDef.Labels
		for val, key := range isbServiceRollout.Spec.InterStepBufferService.Labels {
			newISBServiceDef.Labels[val] = key
		}
```